srcpkgs/seafile-client-qt/template

```diff
@@ -1,20 +1,20 @@
 # Template file for 'seafile-client-qt'
 pkgname=seafile-client-qt
-version=7.0.10
+version=8.0.1
 revision=1
 _sourcename="seafile-client"
 wrksrc="${_sourcename}-${version}"
 build_style=cmake
 configure_args="$(vopt_if shibboleth '-DBUILD_SHIBBOLETH_SUPPORT=ON -DUSE_QT_WEBKIT=ON')"
 hostmakedepends="pkg-config"
-makedepends="qt5-tools-devel seafile-libclient-devel sqlite-devel libevent-devel
+makedepends="qt5-tools-devel seafile-libclient-devel sqlite-devel libevent-devel libsearpc-devel>=3.2.0.20200318
```

Comment:
`libsearpc-devel` dependancy is uneeded since provided via `seafile-libclient-devel`.
version's contrainst is uselees anyway

Reply:
The version constraint was added to prevent attempting to build against an older (unpatched) version of libsearpc, since such a build will fail. The seafile-libclient package will itself build without the patch. Should I remove this constraint anyway?

Reply:
uneeded: runtime constraints are managed either on producing binary package (looks after common/shlibs) and while installing/upgrading.
Since your PR update libsearprc, package upgrade will find it (I've built all this PR locally **without** your constraint "`ibsearpc-devel>=3.2.0.20200318`"):
```
$ sudo xbps-install -Su --repo=/build/packages/hostdir/binpkgs/pr/30384.seafile.8.0.1
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/musl/x86_64-musl-repodata' ...

Name              Action    Version           New version            Download size
libsearpc         update    3.2.0_4           3.2.0.20200318_1       - 
seafile-client-qt update    7.0.10_1          8.0.1_1                - 
seafile-libclient update    7.0.10_2          8.0.1_1                - 

Size required on disk:        4414KB
Space available on disk:      4998MB

Do you want to continue? [Y/n] 
```